internal/commands/pull/command.go

```diff
@@ -84,13 +84,14 @@ func (cmd *Command) Flags() []flags.Flag {
 				},
 			},
 		},
-		flags.StringFlag{
-			Value: &cmd.inputs.TemplateID,
+		flags.StringSliceFlag{
+			Value: &cmd.inputs.TemplateIDs,
 			Meta: flags.Meta{
 				Name:      "template",
 				Shorthand: "t",
 				Usage: flags.Usage{
-					Description: "Specify the Template ID that is used for this Realm app",
+					Description:   "Specify the frontend Template ID(s) to export. Specified templates must be compatible with the remote app",
```

Comment:
what are your thoughts on making the second sentence here a `Note`?  there's not a _ton_ of description/flag usages with periods/multiple sentences yet

Reply:
I'm okay with that but the only this is that the `Note` is not on it's own line.

It seems to be displayed after the `AllowedValues` so it may potentially be lost since the allowed values list is kinda long and not super human readable


![image](https://user-images.githubusercontent.com/15252988/129091593-3ab0da02-f02d-4a14-94a1-184c95550612.png)

Reply:
Having the Note before the Defaults makes sense to me! Given the defaults can get pretty long, getting the Note across first would be good